### ISSUE TITLE
Checkpoint doesn't meet contrast requirements

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#257346]
It should meet the bottom one:

<img width="1510" height="1614" alt="Image" src="https://github.com/user-attachments/assets/3ece7ad0-38cf-4756-9ae1-520cdc0c4626" />

https://webaim.org/resources/contrastchecker/
